R/utils.R

```diff
@@ -317,12 +317,39 @@ un_list <- function(x) {
   names(x) <- nams
   x
 }
-
+# `fmt_txt()` ------------------------------------------------------------------
```

Comment:
I had a few issues with `fmt_txt()` See .Rd for ref. There are duplicated parameters.
https://janmarvin.github.io/openxlsx2/reference/fmt_txt.html has issues too.

There is 
```
#' @param x a string or part of a string
```
but it ends up not appearing anywhere due to duplications of parameters in the same .Rd

Reply:
Not really sure what you mean with "has issues too". You want to avoid the separate `x` and `y` part?

Reply:
I guess, but also, since the rest of the `fmt_txt()` related functions, the base `fmt_txt()` should have its `x` parameter documented as 
```
#' @param x a string or part of a string
```
but since x is later documented in the same rdname otherwise, it is deleted. See `fmt_txt.Rd` to see what I mean.

Reply:
Added a comment in my last commit

Reply:
Got it, well I know who to blame. It's me!